stdlib/public/core/FloatingPoint.swift

```diff
@@ -239,16 +230,6 @@ public protocol FloatingPoint : SignedNumeric, Strideable, Hashable
   ///     the initializer has the same magnitude as `magnitudeOf`.
   init(signOf: Self, magnitudeOf: Self)
 
-% for src_ty in all_integer_types(word_bits):
-  /// Creates a new value, rounded to the closest possible representation.
-  ///
-  /// If two representable values are equally close, the result is the value
-  /// with more trailing zeros in its significand bit pattern.
-  ///
-  /// - Parameter value: The integer to convert to a floating-point value.
-  init(_ value: ${src_ty.stdlib_name})
-
-% end
```

Comment:
Good riddance!

Reply:
I'm *very* curious if, without this, the compiler can optimize the generic initializer down to a single instruction. If so, wow.

Reply:
It very much cannot, but that will be addressed in part two =)